### PR TITLE
Auto OCP-25706

### PR DESCRIPTION
**Pull request by huiran0826**
Auto OCP-25706

Log:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/49250/console

Can you help review?
@weliang1 @zhaozhanqi @anuragthehatter  @rbbratta

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
@huiran0826  how about adding one step (check the pod if work well in that node) after the sdn pod is recreated.

**Comment by huiran0826** [comment]
> @huiran0826 how about adding one step (check the pod if work well in that node) after the sdn pod is recreated.
Thanks for the comments.
Actually restarting  sdn pod is not the required step in this case,  if you think it is better to add a checkpoint for restarting sdn pod, I can add it.

**Comment by huiran0826** [comment]
@zhaozhanqi 
Added another step for restarting sdn pod and checkpoints against the pod works. Can you help review? Thanks!
Here is the new log:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/49675/console

File: features/networking/sdn.feature [hunk 1]
```diff
@@ -258,3 +258,42 @@ Feature: SDN related networking scenarios
...
+    When I run command on the "<%= cb.node_name %>" node's sdn pod:
+      | bash | -c | pgrep ovs-vswitchd \| xargs kill -9 |
+    Then the step should succeed
+    And I wait up to 60 seconds for the steps to pass:
+    """
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-sdn" project
+    And all pods in the project are ready
+    """
+    #Restart sdn pod
```
Comment (zhaozhanqi) [inline features/networking/sdn.feature]: why the SDN pod need to be restarted here?

Comment (huiran0826) [reply features/networking/sdn.feature]: @zhaozhanqi It was from your previous comments required.
@huiran0826 how about adding one step (check the pod if work well in that node) after the sdn pod is recreated.

Comment (zhaozhanqi) [reply features/networking/sdn.feature]: > @zhaozhanqi It was from your previous comments required. Below is previous comments. If you mean sdn pod recreating by killing ovs-vswitchd, actually that will not cause sdn pod restart.
> 
> @huiran0826 how about adding one step (check the pod if work well in that node) after the sdn pod is recreated.
> @zhaozhanqi Thanks for the comments.
> Actually restarting sdn pod is not the required step in this case, if you think it is better to add a checkpoint for restarting sdn pod, I can add it.

oh, misunderstanding this,  I mean check the test pod(ping for pong) if works well after the ovs-vswitch is restarted. 
 BTW, the sdn pod did not be restarted after killing ovs-vswitched ?

File: features/networking/sdn.feature [hunk 2]
```diff
@@ -258,3 +258,42 @@ Feature: SDN related networking scenarios
     When I run cmds on all ovs pods:
       | bash | -c | pgrep ovs-vswitchd |
     Then the step should succeed
+
+  #@author huirwang@redhat.com
```
Comment (liangxia) [inline features/networking/sdn.feature]: Please check the format others are using. Some of auto tools rely on that.

**Comment by huiran0826** [comment]
New logs:
https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/50667/console

File: features/networking/sdn.feature (hunk 2 above)
Comment (huiran0826) [reply features/networking/sdn.feature]: Updated, thanks!

File: features/networking/sdn.feature (hunk 1 above)
Comment (huiran0826) [reply features/networking/sdn.feature]: @zhaozhanqi Updated codes, please help check!

**Comment by huiran0826** [comment]
@pruan-rht  Can you help merge this PR? Thanks!

**Pull request merged**